tests/test_cases/test_discovery/test_discovery.py

```diff
@@ -67,7 +67,7 @@ async def discover_module_values(dut):
         raise TestFailure("Expected to discover things in the DUT")
 
 
-@cocotb.test(expect_error=True)
+@cocotb.test(expect_error=AttributeError)
 async def discover_value_not_in_dut(dut):
     """Try and get a value from the DUT that is not there"""
     fake_signal = dut.fake_signal
```

Comment:
```suggestion
@cocotb.test()
async def discover_value_not_in_dut(dut):
    """Try and get a value from the DUT that is not there"""
    with assert_raises(AttributeError):
        fake_signal = dut.fake_signal
```

Reply:
`assert_raises` isn't available here. Should I copy-paste it?

Reply:
Yeah, I think doing that's fine.